libraries/AP_AccelCal/AccelCalibrator.cpp

```diff
@@ -379,9 +379,6 @@ void AccelCalibrator::run_fit(uint8_t max_iterations, float& fitness)
         }
 
         num_iterations++;
-        if (fitness - last_fitness < 1.0e-9f) {
-            break;
-        }
```

Comment:
isn't this intended so `max_iterations` is interpreted as the max and not the required number of iterations?